packages/front-end/components/Settings/MssqlForm.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC, ChangeEventHandler } from "react";
+import { MssqlConnectionParams } from "back-end/types/integrations/mssql";
+import HostWarning from "./HostWarning";
+import Toggle from "../Forms/Toggle";
+import Tooltip from "../Tooltip";
+
+const MssqlForm: FC<{
+  params: Partial<MssqlConnectionParams>;
+  existing: boolean;
+  onParamChange: ChangeEventHandler<HTMLInputElement>;
+  setParams: (params: {
+    [key: string]: string | boolean | number | Record<string, unknown>;
+  }) => void;
+}> = ({ params, existing, onParamChange, setParams }) => {
+  return (
+    <>
+      <HostWarning
+        host={params.server}
+        setHost={(host) => {
+          setParams({
+            server: host,
+          });
+        }}
+      />
+      <div className="row">
+        <div className="form-group col-md-12">
+          <label>Server</label>
+          <input
+            type="text"
+            className="form-control"
+            name="server"
+            required
+            value={params.server || ""}
+            onChange={onParamChange}
+          />
+        </div>
+        <div className="form-group col-md-12">
+          <label>Port</label>
+          <input
+            type="number"
+            className="form-control"
+            name="port"
+            required
+            value={params.port || ""}
+            onChange={onParamChange}
+          />
+        </div>
+        <div className="form-group col-md-12">
+          <label>Database</label>
+          <input
+            type="text"
+            className="form-control"
+            name="database"
+            required
+            value={params.database || ""}
+            onChange={onParamChange}
+          />
+        </div>
+        <div className="form-group col-md-12">
+          <label>User</label>
+          <input
+            type="text"
+            className="form-control"
+            name="user"
+            required
+            value={params.user || ""}
+            onChange={onParamChange}
+          />
+        </div>
+        <div className="form-group col-md-12">
+          <label>Password</label>
+          <input
+            type="password"
+            className="form-control"
```

Comment:
Change this to match the new datasource password inputs (avoids browser password prompts):
```ts
<input
  type="text"
  className="form-control password-presentation"
```